site/content/en/references/kustomize/kustomization/sortoptions/_index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: "sortOptions"
+linkTitle: "sortOptions"
+type: docs
+weight: 23
+description: >
+    Sort the kustomize resources.
+---
+
+The `sortOptions` field is used to sort the resources kustomize outputs.
+
+IMPORTANT:
+- Currently, this field is enforced only for the top-level kustomization.
+- This is the endorsed way to sort resources. Users should avoid using
+the CLI flag `--reorder` and instead use `sortOptions`.
+
+Currently, we support the following sort options:
+- `legacy`
+- `fifo`
+
+## Legacy Sorting
+
+In `legacy` order, kustomize sorts resources by using two priority lists:
+- An `orderFirst` list for resources which should be first in the output.
+- An `orderLast` list for resources which should be last in the output.
+- Resources not on the lists are will appear in-between, sorted by their GVK.
+
+### Example 1: Legacy Sorting with orderFirst / orderLast lists
+
+In this example, we use the `legacy` sort order to output `Namespace` objects
+first and `Deployment` objects last.
+
+```yaml
+kind: Kustomization
+sortOptions:
+  order: legacy
+  legacySortOptions:
+    orderFirst:
+    - Namespace
+    orderLast:
+    - Deployment
+```
+
+### Example 2: Default Legacy Sorting
+
+If you specify `legacy` sort order without any arguments for the lists,
+kustomize will fall back to the lists we were using before introducing this
+feature. This is useful if you want to continue using the sort order kustomize
+was using by-default before introducing this feature.
```

Comment:
Instead of the last sentence, I'd say something like "Since legacy sort is the default, this is also equivalent to not specifying the field at all."